associated-press-feed/app/config/AWS.scala

```diff
@@ -0,0 +1,28 @@
+package config
+
+import software.amazon.awssdk.auth.credentials.{
+  AwsCredentialsProviderChain,
+  InstanceProfileCredentialsProvider,
+  ProfileCredentialsProvider
+}
+import software.amazon.awssdk.services.s3.S3Client
+import software.amazon.awssdk.regions.Region
+
+object AWS {
+  val defaultRegion: Region = Region.EU_WEST_1
+  val defaultProfile: String = "media-service"
```

Comment:
```suggestion
  val defaultProfile: String = config.getOptional[String]("externalData.profile")
```

Reply:
this doesn't compile, but you make a good point. Perhaps a refactor for a different pr?

Reply:
100 - this is just for illustrative purposes - thanks, will refactor in another PR